tests/integration/test_bibtex_exporting.py

```diff
@@ -22,124 +22,161 @@
-      SLACcitation   = "%%CITATION = HEP-PH/0601012;%%"
-}'''
-    result = Bibtex(proceeding).format()
-
-    assert expected == result
-
-
-@pytest.mark.xfail
-def test_format_thesis(app):
-    thesis = get_db_record('lit', 1395663)
-
-    expected = u'''@phdthesis{Mankuzhiyil:2010jpa,
-      author         = "Mankuzhiyil, Nijil",
-      title          = "{MAGIC $\\gamma$-ray observations of distant AGN and a study of source variability and the extragalactic background light using FERMI and air Cherenkov telescopes}",
-      school         = "Udine U.",
-      year           = "2010",
-      url            = "https://magicold.mpp.mpg.de/publications/theses/NMankuzhiyil.pdf",
-      SLACcitation   = "%%CITATION = INSPIRE-1395663;%%"
-}'''
-    result = Bibtex(thesis).format()
-
-    assert expected == result
-
-
-@pytest.mark.xfail
+    proceedings = get_es_record('lit', 701585)
+    expected = ("Alekhin:2005dx", Entry('proceedings', [
+        ('key', '701585'),
+        ('address', u"Geneva"),
+        ('pages', u"pp.1--326"),
+        ('publisher', u"CERN"),
+        ('title', u"HERA and the LHC: A Workshop on the implications of HERA for LHC physics: Proceedings Part A"),
+        ('year', u"2005"),
+        ('reportNumber', u"CERN-2005-014, DESY-PROC-2005-01"),
+        ('archivePrefix', u"arXiv"),
+        ('eprint', u"hep-ph/0601012"),
+        ('primaryClass', u"hep-ph"),
+        ('SLACcitation', u"%%CITATION = HEP-PH/0601012;%%"),
+        ('url', u"http://weblib.cern.ch/abstract?CERN-2005-014"),
+    ], persons={
+        'editor': [Person(u"De Roeck, A."), Person(u"Jung, H.")],
+        'author': [],
+    }))
+
+    schema = PybtexSchema()
+    result = schema.load(proceedings)
+
+    assert result is not None
+    assert pybtex_entries_equal(result, expected)
+
+
+def test_format_phdthesis(app):
+    phdthesis = get_es_record('lit', 1395663)
+    expected = ("Mankuzhiyil:2010jpa", Entry('phdthesis', [
+        ('key', '1395663'),
+        ('school', u"Udine U."),
+        ('title', u"MAGIC $\\gamma$-ray observations of distant AGN and a study of source variability and the extragalactic background light using FERMI and air Cherenkov telescopes"),
+        ('year', u"2010"),
+        ('url', u"https://magicold.mpp.mpg.de/publications/theses/NMankuzhiyil.pdf"),
+    ], persons={
+        'editor': [],
+        'author': [Person(u"Mankuzhiyil, Nijil")],
+    }))
+
+    schema = PybtexSchema()
+    result = schema.load(phdthesis)
+
+    assert result is not None
+    assert pybtex_entries_equal(result, expected)
+
+
 def test_format_book(app):
-    book = get_db_record('lit', 736770)
+    book = get_es_record('lit', 736770)
+    expected = ("Fecko:2006zy", Entry('book', [
+        ('key', '736770'),
+        ('publisher', u"Cambridge University Press"),
+        ('title', u"Differential geometry and Lie groups for physicists"),
+        ('year', u"2011"),
+        ('isbn', u"9780521187961, 9780521845076, 9780511242960"),
```

Comment:
you should try to normalize the isbn in order to put back the dashes (it might fail as some are invalid).